### PR TITLE
Landing Page, Category and Rules added in mini project.

### PR DESCRIPTION
**Pull request by ashifkhn**
Landing Page, Category and Rules added in mini project.
Inspect the deploy log: https://app.netlify.com/sites/xenodochial-kirch-e898c8/deploys/621158649c892a0009651e7a
Browse the preview: https://deploy-preview-1--xenodochial-kirch-e898c8.netlify.app/

### CODE REVIEW

File: components/category/category.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  flex-direction: column;
+  height: 100%;
+}
+
+nav {
+  display: flex;
+  /* width: 100%; */
+  justify-content: space-between;
+  text-align: center;
+  color: #414141;
```
Comment (vikrantkode) [inline components/category/category.css:16]: use variable to define colors wherever necessary

File: components/rules/rules.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+.rulesContainer {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+
+  padding: 5rem;
+  color: #ffe6e6;
+}
```
Comment (vikrantkode) [inline components/rules/rules.css:41]: same here

File: css/background.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/* .colorPrimary {
+  background-color: #e27d60;
+} */
+
+.colorPrimary {
+  background-color: #7dc3ae;
+}
```
Comment (vikrantkode) [inline css/background.css:7]: same here for variables

File: css/fontSize.css [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (vikrantkode) [inline css/fontSize.css:1]: cant see any css here

**Review by vikrantkode — COMMENTED** [review]
- some changes suggested
- Use single component for 1 pr... will be easy to preview
- Rest good to go..

**Review by kirti2602 — APPROVED** [review]
Good to go. Just try using rems instead of px

File: css/background.css (hunk 3 above)
Comment (ashifkhn) [reply css/background.css:7]: done.

**Pull request merged**